tox.ini

```diff
@@ -28,6 +28,7 @@ commands=
     native-blockchain-spurious_dragon: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork EIP158}
     native-blockchain-byzantium: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork Byzantium}
     native-blockchain-constantinople: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork Constantinople}
+    native-blockchain-petersburg: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork ConstantinopleFix}
```

Comment:
Hm, seems unfortunate to call it petersburg in one place and ConstantinopleFix in another. Obviously this isn't the first fork to have that problem, but I can't think of a good reason we'd want to continue the divergence.

Reply:
It's unfortunate that we didn't manage to reach consensus among all dev teams how to name this fork. Geth and Py-EVM both call it Petersburg but `ethereum/tests` calls it `ConstantinopleFix` which is why we have `ConstantinopleFix` here. I believe this should be fixed upstream because `ConstantinopleFix` imho is visually too close to `Constantinople` and makes `Constantinople` less grepable.

Reply:
cc @holgerd77 and @winsvega what do you two think about changing the name.

@cburgdorf you might surface this in the `ethereum/tests` repo as an issue

Reply:
I'd even prefer calling the old one `ConstantiNOPEl` and the new one `Constantinople` :smile: -- caps make it visually apparent, and it's greppable

Reply:
@carver I remember I proposed in the AllCoreDevs that we should consider renaming the failed fork to free up the name for the new fork but there were concerns that we can't do that since it is already live on various nets including xDai. Technically, we can do whatever we want in this code base but I guess deciding on a name in `ethereum/tests` opens up a broader discussion again ;)

Reply:
If the issue is the CLI arg being named as-upstream instead of as-codebase, could there be a "convenience" mapping of names for the `--fork` arg?..

Reply:
@veox we could do that but it just moves the `ConstantinopleFix` to another place. With geth, EthereumJS, Py-EVM all using `Petersburg` as a name, maybe an upstream change would be appropriate.